README.md

```diff
@@ -0,0 +1,3 @@
+# Custom Disk Image for virtual machines
+
+For detailed instructions and best practices, we recommend referring to the official [CheerpX documentation](https://cheerpx.io/docs/overview). It contains all the necessary information to help you get started effectively.
```

Comment:
Please point people directly to the custom image page and remove the build.sh file